.claspignore

```diff
@@ -1,4 +1,5 @@
 .git/**
 .idea/**
 build/**
-node_modules/**
\ No newline at end of file
+node_modules/**
+__tests__/**
```

Comment:
I think it would be easier to find the test file by putting them along with the production file.
And normally those test files are named as `NAME.test.ts`. And you could filter them like `*.test.*` here.

Reply:
Good call much nicer.